Sources/TeslaSwift/Model/EnergySiteInfo.swift

```diff
@@ -23,8 +23,8 @@ open class EnergySiteInfo: Codable {
     open var offGridVehicleChargingReservePercent: Double?
     
     open var userSettings: UserSettings
-    open var touSettings: TOUSettings?
```

Comment:
`touSettings` arent there anymore?
what about for WallConnector?

Reply:
Sorry I was on a code sabbatical. I'm back at my Swift efforts now.  I do not have a wall connector so maybe my JSON is no the same as yours? Can you give me a sample JSON for an energy site with a wall connector?

Reply:
made a commit with the missing fields